Cargo.toml

```diff
@@ -17,6 +17,13 @@ include = ["src/*", "Cargo.toml", "LICENSE", "README.md"]
 bitflags = "1.3"
 byteorder = "1.4"
 flate2 = "1"
+tokio = { version = "1.21", default-features = false, features = ["io-util", "fs"], optional = true}
 
 [dev-dependencies]
 tempfile = "3"
+tokio = { version = "1.21", default-features = false, features = ["rt", "macros", "io-util", "fs"]}
```

Comment:
This seems pretty gross, but I'm not sure how to effectively have tokio in tests otherwise.  Could have everyone remember to run with `--all-features`, or could make `tokio` enabled by default and switch the feature flag to disable.